.github/workflows/scripts/linux_arm64/rh8/test_wheels.sh

```diff
@@ -12,16 +12,8 @@ export PYTHON_PACKAGE=$4
 # Install system dependencies (gcc gives access to c99, which is needed by some tests)
 dnf update -y 
 dnf install -y libzstd-devel gcc-toolset-${GCC_VERSION} gcc
-if [ "$PYTHON_VERSION" != "3.10" ]; then
-    dnf install -y ${PYTHON_PACKAGE} ${PYTHON_PACKAGE}-devel
-fi
```

Comment:
Is this going to work? I thought we might want to test removing the `if` guard 😅

Reply:
My bad. I deleted the installation part by mistake.